docs/titan-reference/button-reference.md

```diff
@@ -0,0 +1,485 @@
+
+ --- 
+## \<Go\> 
+ Runs the next cue of the connected playback, usually used when running cues from a cue list. See [Running a Cue List](../cue-lists/cue-list-playback.md/#running-a-cue-list).
+
+ --- 
+## \<Go page\> 
+ Type a number to jump to Page ``<n>`` of the associated fader bank (consoles with multiple fader banks may have several of these buttons). See also \<Page +\> and \<Page -\>. See [Changing playback pages](../cues/cue-playback.md/#changing-playback-pages).
+
+ --- 
+## \<Gobo\> 
+ Selects the Gobo attribute group so the wheels control gobo select, rotation etc. Also used for creating masks. See [Changing Attributes using the Wheels](../controlling-fixtures/changing-fixture-attributes.md/#changing-attributes-using-the-wheels).
+
+ --- 
+## \<Group\> 
+ Opens the group menu for saving and editing fixture groups. See [Fixture Groups](../controlling-fixtures/fixture-groups.md/#using-fixture-groups).
+
+ --- 
+## \<Hi light\> 
+ Starts or ends Hi-light mode which visually shows the selected fixture(s) on stage by increasing intensity or setting a contrasting colour. Often used with \<Fix +1\> and \<Fix -1\> buttons. See [Stepping through selected fixtures](../controlling-fixtures.md/#stepping-through-selected-fixtures-one-at-a-time).
+
+ --- 
+## \<Include\> 
+ (followed by playback Select) Loads the contents of a playback back into the programmer for further editing or re-use. See [Include Function](../cues/editing-cues.md/#using-parts-of-existing-cues---the-include-function).
+
+ --- 
+## \<Intensity\> 
+ Selects the Intensity attribute group so the wheels control Dimmer, Shutter etc. Also used for creating masks. See [Changing Attributes using the Wheels](../controlling-fixtures/changing-fixture-attributes.md/#changing-attributes-using-the-wheels).
+
+ --- 
+## \<Keyboard\> 
+ (D9 only) Shows the touch screen keyboard. On other consoles there is a button in the window title bars to open the keyboard. See [Touch Keyboard](../titan-basics/workspace-windows.md/#touch-keyboard).
+
+ --- 
+## \<Latch\> 
+ (D9 only) Latches the current menu so that it remains active when you complete the action. \<Latch Menu\> on other consoles, see below.
+
+ --- 
+## \<Latch Menu\> 
+ Latches the current menu so that it remains active when you complete the action. Useful for copy, move etc. Useful for copy, move etc. See [The Menu/Softkeys](../titan-basics/other-parts-of-the-touch-screen.md/#the-menusoftkeys).
+
+ --- 
+## \<Left\> 
+ (D9 / ST only) Left mouse button when trackball is in mouse mode.
+
+ --- 
+## \<Legend\> 
+ (D9 only) (followed by a Select button) Set legend on an item. On other consoles use the \[Set Legend\] softkey at the top level menu. See [Legends and picture legends](../titan-basics/workspace-windows.md/#legends-and-picture-legends).
+
+ --- 
+## \<Levels\> 
```

Comment:
Maybe it would be worth indicating all of the wheel paging buttons in a similar way: `<Levels> (wheel paging)`